<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculatrice à Notation Polonaise Inversée - Amhed &amp; JB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineSaisie">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="QTabWidget" name="tabWidget_pile">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Pile 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListView" name="listView_pile"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Pile 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListView" name="listView_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="paramSaisie">
      <item>
       <widget class="QCheckBox" name="checkBox_complexe">
        <property name="text">
         <string>Complexe</string>
        </property>
        <property name="shortcut">
         <string>F1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_reel">
        <property name="text">
         <string>Reel</string>
        </property>
        <property name="shortcut">
         <string>F2</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_rationnel">
        <property name="text">
         <string>Rationnel</string>
        </property>
        <property name="shortcut">
         <string>F3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_entier">
        <property name="text">
         <string>Entier</string>
        </property>
        <property name="shortcut">
         <string>F4</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_calculAuto">
        <property name="text">
         <string>Calcul auto</string>
        </property>
        <property name="shortcut">
         <string>'</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>1000</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget_saisie">
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_basique">
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="title">
        <string>Basiq&amp;ue</string>
       </attribute>
       <attribute name="toolTip">
        <string>Pavé numérique et de calcul de base
Raccourci : U
NB: les boutons sont perpétuellement rattachés aux touches de clavier correspondantes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_0">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_point">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_space">
          <property name="text">
           <string>space</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_diviser">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButton_plus">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_moins">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButton_fois">
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_divers">
       <attribute name="title">
        <string>D&amp;ivers</string>
       </attribute>
       <attribute name="toolTip">
        <string>Diverses fonctions mathématiques
Raccourci : I</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_pow">
          <property name="toolTip">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Dépile X puis Y et effectue Y puissance X&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;X: Entier, Rationnel, Réel&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;Y: Entier, Rationnel, Réel&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;Rac.: **&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>POW</string>
          </property>
          <property name="shortcut">
           <string>P</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_inv">
          <property name="text">
           <string>INV</string>
          </property>
          <property name="shortcut">
           <string>I</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_mod">
          <property name="text">
           <string>MOD</string>
          </property>
          <property name="shortcut">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_sqrt">
          <property name="toolTip">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Dépile X et rempile sa racine carrée&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;X: Entier, Rationnel, Réel&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;Rac.: &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;^&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;+&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;/&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>SQRT</string>
          </property>
          <property name="shortcut">
           <string>Q</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_sign">
          <property name="text">
           <string>SIGN</string>
          </property>
          <property name="shortcut">
           <string>_</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_fact">
          <property name="text">
           <string>FACT (!)</string>
          </property>
          <property name="shortcut">
           <string>!</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_sqr">
          <property name="text">
           <string>SQR</string>
          </property>
          <property name="shortcut">
           <string>²</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_ln">
          <property name="text">
           <string>LN</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_eval">
          <property name="text">
           <string>EVAL</string>
          </property>
          <property name="shortcut">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_cube">
          <property name="text">
           <string>CUBE</string>
          </property>
          <property name="shortcut">
           <string>U</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_log">
          <property name="text">
           <string>LOG</string>
          </property>
          <property name="shortcut">
           <string>O</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_trigo">
       <attribute name="title">
        <string>Trig&amp;o</string>
       </attribute>
       <attribute name="toolTip">
        <string>Fonctions trigonométriques
Raccourci : O</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_sin">
          <property name="text">
           <string>SIN</string>
          </property>
          <property name="shortcut">
           <string>S</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButton_radian">
          <property name="text">
           <string>Radian</string>
          </property>
          <property name="shortcut">
           <string>R</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_cos">
          <property name="text">
           <string>COS</string>
          </property>
          <property name="shortcut">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="radioButton_degre">
          <property name="text">
           <string>Degré</string>
          </property>
          <property name="shortcut">
           <string>D</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_tan">
          <property name="text">
           <string>TAN</string>
          </property>
          <property name="shortcut">
           <string>T</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBox_hyperbolic">
          <property name="text">
           <string>Hyperbolic</string>
          </property>
          <property name="shortcut">
           <string>H</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_pile">
       <attribute name="title">
        <string>&amp;Pile</string>
       </attribute>
       <attribute name="toolTip">
        <string>Fonctions pour agir sur la pile.
Raccourci : P</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_swap">
          <property name="text">
           <string>SWAP</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+\</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_clear">
          <property name="text">
           <string>CLEAR</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Del</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_sum">
          <property name="text">
           <string>SUM</string>
          </property>
          <property name="shortcut">
           <string>Ctrl++</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_dup">
          <property name="text">
           <string>DUP</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_mean">
          <property name="text">
           <string>MEAN</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+/</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_drop">
          <property name="text">
           <string>DROP</string>
          </property>
          <property name="shortcut">
           <string>Del</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_0</sender>
   <signal>clicked()</signal>
   <receiver>tabWidget_pile</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_point</sender>
   <signal>clicked()</signal>
   <receiver>tabWidget_pile</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
